wolfSSL/src/chapter02.md

```diff
@@ -394,6 +394,122 @@ Removes code for OAEP padding.
 
 Turns off AES-CBC algorithm support.
 
+#### WOLFCRYPT_ONLY
+
+Enables wolfCrypt only while disabling TLS.
+
+#### NO_CAMELLIA_CBC
+
+Disables Camellia CBC support.
+
+#### NO_AES
+
+Disables AES algorithm support.
+
+#### NO_AESGCM_AEAD
+
+Disables AES GCM algorithm support.
+
+#### NO_ASN_TIME
+
+Disables time checking for ASN.
+
+#### NO_CHECK_PRIVATE_KEY
+
+RSA - needed to encrypt salt.
```

Comment:
This macro disables additional private key checking that is on by default. This enables checking to validate the private key is a pair for the public key. It is supported for RSA, ECDSA, ED25519, ED448, Falcon, Dilithium and Sphincs.